src/sinopiaApi.js

```diff
@@ -11,14 +11,21 @@ import {
 } from "../__tests__/testUtilities/fixtureLoaderHelper"
 import GraphBuilder from "GraphBuilder"
 import { v4 as uuidv4 } from "uuid"
-import Auth from "@aws-amplify/auth"
```

Comment:
this removal surprises me, but I trust you.

Reply:
It was moved to a helper module.,